tools/ci_build/github/azure-pipelines/templates/set-version-number-variables-step.yml

```diff
@@ -0,0 +1,32 @@
+# Sets version number from VERSION.txt into a variable. As well as the git commit hash.
+
+steps:
+- task: CmdLine@2
+  displayName: 'Set Version Number variables for  Windows'
+  inputs:
+    script: |
+        SETLOCAL EnableDelayedExpansion
+        set /p _OnnxRuntimeVersion=<$(Build.SourcesDirectory)\VERSION_NUMBER 
+        @echo ##vso[task.setvariable variable=OnnxRuntimeVersion;]%_OnnxRuntimeVersion%
+        
+        FOR /F "tokens=* USEBACKQ" %%F IN (`git rev-parse --short HEAD`) DO (
```

Comment:
This gives the short version of the commit hash. Do we want to retain the full hash instead? Or is short version sufficient?

Reply:
if kept in an internal file, will use the long commit hash